### ISSUE TITLE
Deprecation warnings (due until 2025.1)

### ISSUE THREAD

**Issue by R1DEN** [barleybobs/homeassistant-ecowater-softener#50]
Got this error message in the logs
```
Logger: homeassistant.const
Source: helpers/deprecation.py:204
First occurred: 4 January 2024 at 09:05:46 (5 occurrences)
Last logged: 4 January 2024 at 09:05:57

VOLUME_LITERS was used from ecowater_softener, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfVolume.LITERS instead, please create a bug report at https://github.com/barleybobs/homeassistant-ecowater-softener/issues
VOLUME_GALLONS was used from ecowater_softener, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfVolume.GALLONS instead, please create a bug report at https://github.com/barleybobs/homeassistant-ecowater-softener/issues
TIME_DAYS was used from ecowater_softener, this is a deprecated constant which will be removed in HA Core 2025.1. Use UnitOfTime.DAYS instead, please create a bug report at https://github.com/barleybobs/homeassistant-ecowater-softener/issues
```

**Comment by figorr** [comment]
I found the same issue when I updated HA Core to 2024.1 yesterday.

I sent a pull request in order to modify the sensor.py file to try to solve this.

[https://github.com/barleybobs/homeassistant-ecowater-softener/pull/49/files](url)

I have no time to test it yet, because I am outside home and I cannot access through ssh. But if you want ... you can make a backup of your sensor.py file, just for there is an error and you need to recover it. Then you can edit the sensor.py file and restart Home Assistant to apply changes. It should work ... but if there is any error ... you can always recover your sensor.py backup file ... and restart Home Assistant to apply the changes.

**Comment by barleybobs** [comment]
This has been fixed by pull request #49.

Thanks 👍
